docs/layouts/shortcodes/generated/auto_scaler_configuration.html

```diff
@@ -21,10 +21,16 @@
             <td>Enable job autoscaler module.</td>
         </tr>
         <tr>
-            <td><h5>kubernetes.operator.job.autoscaler.metrics.window</h5></td>
+            <td><h5>kubernetes.operator.job.autoscaler.metrics.window.size.max</h5></td>
```

Comment:
could also be simply `...metrics.window.max` and `...metrics.window.min` to simplify naming